cassdegrees/ui/views/student.py

```diff
@@ -49,6 +49,13 @@ def load_messages(cookies):
             del cookies['message']
         except KeyError:
             pass
+    popup = cookies.get('popup', None)
```

Comment:
Do we need to use cookies for this one? This seems a bit round-about. It is possible to do this clientside?

Reply:
Cookies aren't needed, although I don't know any other way to do this without either using cookies or displaying it in the URL parameters, and I feel this way is a bit cleaner on the user-end